### ISSUE TITLE
"Error: No such file or directory (os error 2)" when running the server

### ISSUE THREAD

**Issue by ditod** [EricLBuehler/mistral.rs#223]
**Describe the bug**
When running the server after a fresh clone and build

**Latest commit**
commit 092deeec5ed9c45b36df280d6eba2b0632d4f415

**How to reproduce**
```shell
git clone https://github.com/EricLBuehler/mistral.rs.git
cd mistral.rs
cargo run -- -i plain -m microsoft/Phi-3-mini-128k-instruct -a phi3
```

hint: I think the problem might be with huggingface integration because when i run the server in the model dir with local as `mistralrs-server --port 1234 plain -m . -a phi3` it works fine.

**Comment by EricLBuehler** [comment]
@ditod, was this fixed by #225?

**Comment by EricLBuehler** [comment]
Please feel free to re-open!
